.gitignore

```diff
@@ -11,3 +11,8 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 coverage.txt
+
+# VIM swap files
+*~
+*.swp
+*.swo
```

Comment:
please put this in you global `~/.gitignore` (editor specific setup doesn't belong with project).

Reply:
@juliaogris While trying to correct this PR with git rebase, I think I have made a mess of this PR. I am reverting someone's changes as well. Now I am not sure how to fix this PR. Shall I discard this PR and open a new one in a new branch?